src/apps/sommelier/sommelierApi.ts

```diff
@@ -0,0 +1,86 @@
+import { Address } from 'viem'
+import { NetworkId } from '../../types/networkId'
+import got from '../../utils/got'
+
+const NETWORK_ID_TO_SOMM_CHAIN: Record<NetworkId, string | null> = {
+  [NetworkId['celo-mainnet']]: null,
+  [NetworkId['ethereum-mainnet']]: 'ethereum',
+  [NetworkId['arbitrum-one']]: 'arbitrum',
+  [NetworkId['op-mainnet']]: 'optimism',
+  [NetworkId['polygon-pos-mainnet']]: null,
+  [NetworkId['base-mainnet']]: null,
+  [NetworkId['celo-alfajores']]: null,
+  [NetworkId['ethereum-sepolia']]: null,
+  [NetworkId['arbitrum-sepolia']]: null,
+  [NetworkId['op-sepolia']]: null,
+  [NetworkId['polygon-pos-amoy']]: null,
+  [NetworkId['base-sepolia']]: null,
+}
+
+export async function getSommStrategiesData(networkId: NetworkId) {
+  const { result } = await got
+    .get(`https://app.somm.finance/api/sommelier-api-all-strategies-data`)
+    .json<{
+      result: {
+        data: {
+          cellars: {
+            id: Address
+            shareValue: string
+            tvlTotal: number
+            chain: string
+          }[]
+        }
+      }
+    }>()
+
+  return result.data.cellars
+    .filter((cellar) => cellar.chain === NETWORK_ID_TO_SOMM_CHAIN[networkId])
+    .map((cellar) => {
+      const cellarAddress = cellar.id.split('-')[0] as Address // some cellars have a suffix to the address (e.g. `-arbitrum`), which we do not need
+      return {
+        ...cellar,
+        address: cellarAddress,
```

Comment:
nit: So we don't need to lowercase further down the line.
```suggestion
        address: cellarAddress.toLowerCase(),
```